app/templates/components/explore/side-bar.hbs

```diff
@@ -0,0 +1,105 @@
+<div class="ui list">
```

Comment:
@niranjan94 The DOM got cluttered because I removed the accordions. And temporarily added lists to maintain the uniformity, this was in no way the final proposed DOM structure. Apologies for the misunderstanding.

Yes, that seems great, just one thing. The menus will be always be in the expanded state? Like in the fiddle?